src/solver/infeasible-problem-analysis/constraint-slack-analysis.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "unfeasibility-analysis.h"
+
+namespace Antares::Optimization
+{
+
+class ConstraintSlackAnalysis : public UnfeasibilityAnalysis
+{
+    using UnfeasibilityAnalysis::UnfeasibilityAnalysis;
+public:
+    void run() override;
+    void printReport() override;
+    std::string title() override { return "Slack variables analysis"; }
+
+private:
+    void buildObjective() const;
+    void addSlackVariables();
+    operations_research::MPSolver::ResultStatus Solve() const;
+
+    std::vector<const operations_research::MPVariable*> slackVariables_;
+    const std::string constraint_name_pattern = "^AreaHydroLevel::|::hourly::|::daily::|::weekly::|^FictiveLoads::";
```

Comment:
Should be `constexpr`

Reply:
Impossible to associate **constexpr** to a string, compilation refuses. Even **static const** does not compile.